CONTRIBUTING.md

```diff
@@ -80,15 +70,24 @@ Must be one of the following:
 * **style**: Changes that do not affect the meaning of the code (white-space, formatting, etc)
 * **refactor**: A code change that neither fixes a bug nor adds a feature
 * **perf**: A code change that improves performance
+* **revert**: Reverts a previous commit
 * **test**: Adding missing tests
 * **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
   generation
 
-### Scope
+##### Revert
+If the commit reverts a previous commit, it should begin with `revert: `, followed by the header of the reverted commit. In the body it should say: `Reverts commit <hash>.`, where the hash is the SHA of the commit being reverted.
```

Comment:
Idem ^. This change should be made to comply with qTox own rules, I will test it (I made a typo here, should be `revert`).